jax/_src/numpy/ufuncs.py

```diff
@@ -3150,9 +3150,36 @@ def hypot(x1: ArrayLike, x2: ArrayLike, /) -> Array:
   return _where(idx_inf, _lax_const(x, np.inf), x)
 
 
-@implements(np.reciprocal, module='numpy')
 @partial(jit, inline=True)
 def reciprocal(x: ArrayLike, /) -> Array:
+  """Calculate element-wise reciprocal of the input.
+
+  JAX implementation of :obj:`numpy.reciprocal`.
+
+  The reciprocal is calculated by ``1/x``.
+
+  Args:
+    x: input array or scalar.
+
+  Returns:
+    An array of same shape as ``x`` containing the reciprocal of each element of
+    ``x``.
+
+  Note:
+    - ``jnp.reciprocal`` returns the reciprocal of the integer inputs, unlike
+      ``numpy.reciprocal`` which returns ``0`` for all the integer inputs.
```

Comment:
I think this is not strictly true: for `0` and `1` `np.reciprocal` does not return ``0``.

Maybe a better way to say this is that for integer inputs, `np.reciprocal` returns rounded integer output, while `jnp.reciprocal` promotes integer inputs to floating point.

Reply:
Thanks! Updated as suggested